scst-store/failover.md

```diff
@@ -0,0 +1,20 @@
+# Failover, redundancy, and backups
+
+### API Server
+
+By default the API server only has 1 replica. 
+If the pod crashes, the single instance should restart by normal kubernetes behaviour, but there will be downtime.
+If the user is upgrading, some downtime is expected as well.
+
+Users have the option to configure the number of replicas using the `app_replicas` field in the `scst-store-values.yaml` file.
+
+### Database
+
+By default, the database only has 1 replica, and would restart with some downtime if it were to crash.
+
+Although the field `db_replicas` exists and is configurable by the user in the `scst-store-values.yaml` file, it is experimental and is not recommended to be used.
```

Comment:
I still don't understand what it means to set the database replica to something higher than 1. Doesn't that mean that the app deployment is communicating interchangeably between two databases randomly?

Reply:
yes, but also they are would be attached to the same volume which will likely corrupt the database

Reply:
I'm not sure it's something we should even be telling people about. It seems like a good way to teach people how to shoot themselves in the foot. Like I'm concerned that it might make people think that our app deployment supports multiple databases to help with scaling or something like that. 

Maybe we can be more explicit and say something like the `internal_db` should not be used in production?

Reply:
they can see that the `db_replicas` field is available to them though,  isn't it better to call out here that even though it exists we don't recommend them to use it? Unless we remove that field altogether. I can change the wording to be "Although the field `db_replicas` exists and is configurable by the user in the `scst-store-values.yaml` file, we do not recommend using it. The default internal db should not be used in production." Is that better?

Reply:
I am ok with the wording. The field can still be configured we should warn them not to use it. It's also a good call out not to use the internal db for production use.